clouddrift/dataformat.py

```diff
@@ -24,11 +24,16 @@ def __init__(
         self.validate_attributes()
 
     @classmethod
-    def from_awkward(cls, array: ak.Array):
+    def from_awkward(
+        cls,
+        array: ak.Array,
+        vars_coords: Optional[list] = ["time", "lon", "lat", "ids"],
+    ):
         """Load a RaggedArray instance from an Awkward Array.
 
         Args:
             array (ak.Array): Awkward Array instance to load the data from
+            name_coords (list, optional): Names of the coordinate variables in the ragged arrays
```

Comment:
`name_coords` or `vars_coords`? I suggest just `coords`.

Reply:
But there is already a `coords` in RaggedArray which is a dictionary containing the data, so I think it would be better to avoid having the same variables.

My choice would be to rename the `vars_` -> `name_`, but I'm open to suggestions.

Reply:
Ah OK, that's fine then. In that case let's just make the docstring consistent with the arguments list.